<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Core::SettingsPages::MainWindowSettingsWidget</class>
 <widget class="QWidget" name="Core::SettingsPages::MainWindowSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>333</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="displayGroupBox">
     <property name="title">
      <string/>
     </property>
     <layout class="QFormLayout" name="_2">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <item row="0" column="0" colspan="2">
       <widget class="QCheckBox" name="alwaysOnTopCheckBox">
        <property name="text">
         <string>Always on top</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="hideOnStartupCheckBox">
        <property name="text">
         <string>Hide on startup</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="hideFromTaskbarCheckBox">
        <property name="text">
         <string>Hide from taskbar</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="autoHideLabel">
        <property name="text">
         <string>Autohide after (0 = disabled):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="2">
       <widget class="QSlider" name="autoHideSlider">
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="autoHideMinLabel">
        <property name="text">
         <string>0 sec</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLabel" name="autoHideMaxLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>120 sec</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="transparencyLabel">
        <property name="text">
         <string>Transparency:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QSlider" name="transparencySlider">
        <property name="minimum">
         <number>10</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="transparencyMinLabel">
        <property name="text">
         <string notr="true">10%</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLabel" name="transparencyMaxLabel">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true">100%</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="dockingGroupBox">
     <property name="title">
      <string>Docking</string>
     </property>
     <layout class="QFormLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::ExpandingFieldsGrow</enum>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="dockDistanceLabel">
        <property name="text">
         <string>When closer than:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="dockDistanceSpinBox">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> px</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QCheckBox" name="dockToEdgesCheckBox">
        <property name="text">
         <string>Dock to desktop edges</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dockToEdgesCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>dockDistanceSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>295</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
